spec/features/courses/index_spec.rb

```diff
@@ -62,9 +61,17 @@
         expect(find('[data-qa="courses-table__course"]')).to have_content(course_3.attributes[:name])
       end
     end
+
+    scenario "it shows 'add a new course' link" do
```

Comment:
Maybe overkill but there can be 2 'add a new course' links

Reply:
Overkill.